crates/pipeline_manager/src/db/mod.rs

```diff
@@ -28,10 +28,14 @@ pub(crate) mod storage;
 /// time, which determines the position of the project in the queue.
 pub(crate) struct ProjectDB {
     conn: Client,
-    #[allow(dead_code)] // We don't have to interact with it, but dropping it stops the DB server.
-    pub pg: Option<pg_embed::postgres::PgEmbed>,
 }
 
+// Used in dev mode for having an embedded Postgres DB live through the lifetime
+// of the program.
+#[cfg(feature = "dev")]
+static PG: once_cell::sync::OnceCell<pg_embed::postgres::PgEmbed> =
```

Comment:
If the program crashes will it call `drop()` on PgEmbed (and hence stop Postgres) if it runs in once_cell? I think it's wasy to test by running it an hitting ctrl+c or kill-9. I remember it was a struggle getting it to call drop with tokio once cell stuff in tests, but maybe a regular once_cell is fine.

Reply:
I suspect all bets are off when we call kill-9 unless we trap the signal?

Reply:
I don't think you can trap SIGKILL, only SIGTERM

Reply:
@gz It's better to not rely on drop for this. Currently, the pipeline manager doesn't restart correctly because the DDL statements being issued are not idempotent. The fix for that is to do a pass on the schema as we discussed (some of the 1:1 mappings should be a single table).

Reply:
you're right about SIGKILL, but I'm wondering if OnceCell content gets dropped if the user hits ctrl+c (otherwise there will be a lingering postgres process). Just mentioned because I think for the tests I had to put it in a struct member to get drop() called.

I can fix the 1:1 mappings as once the webui stuff is merged.

Reply:
I tried it; looks like drop gets called on the OnceCell.